docs/guides/sdks/ios-sdk/integrate-ios-sdk.md

```diff
@@ -251,7 +251,9 @@ To set the CUID:
 AppsFlyerLib.shared().customerUserID = "my user id"
 ```
 
-
+> 📘 Note
+> 
+> Since the Customer User ID must be set with every app launch, ensure it is set within the `applicationDidBecomeActive` lifecycle method.
```

Comment:
@dori-af There is no `applicationDidBecomeActive` lifecycle method in iOS.
I would leave it with every app launch.